Sources/CocoaLumberjack/DDFileLogger.m

```diff
@@ -220,11 +220,11 @@ - (void)deleteOldLogFiles {
         for (NSUInteger i = firstIndexToDelete; i < sortedLogFileInfos.count; i++) {
             DDLogFileInfo *logFileInfo = sortedLogFileInfos[i];
 
-            __autoreleasing NSError *error;
+            __autoreleasing NSError *error = nil;
             BOOL success = [[NSFileManager defaultManager] removeItemAtPath:logFileInfo.filePath error:&error];
             if (success) {
                 NSLogInfo(@"DDLogFileManagerDefault: Deleting file: %@", logFileInfo.fileName);
-            } else if (error) {
```

Comment:
IMHO this wasn't entirely wrong. Since we don't really handle the error, it seems safer to also check it before logging it.